content/docs/started/getting-started.md

```diff
@@ -1,109 +1,177 @@
+	 models.
+
+This list is based ONLY on current capabilities. We are investing significant
+resources to expand the functionality and actively soliciting help from
+companies and individuals interested in contributing (see
+[Contributing](/docs/contributing/)).
+
+## Installation without pre-existing Kubernetes
+
+Here is how you can get Kubeflow up and running if you don't have a K8s cluster
+running already.
+
+### Local
+
+There are a several options:
+
+   * [MiniKF](/docs/started/getting-started-minikf/)
+      * MiniKF is a fast and easy way to get started with Kubeflow.
+      * It installs with just two commands and then you are up for
+	experimentation, and for running complete Kubeflow Pipelines.
+      * MiniKF runs on all major operating systems (Linux, macOS, Windows)
+
+   * [Minikube setup](/docs/started/getting-started-minikube/)
+      * Minikube leverages virtualization applications like [Virtual
+	Box](https://www.virtualbox.org/) or [VMware
+	Fusion](https://www.vmware.com/products/fusion.html) to host the virtual
+	machine and provides a CLI that can be leveraged outside of the VM.
+      * Minikube defines a fully baked ISO that contains a minimal operating
+	system and kubernetes already installed.
+      * This option may be useful if you are just starting to learn and already
+	have one of the virtualization applications already installed.
+
+   * [Microk8s setup](/docs/started/getting-started-multipass/)
+      * The benefits of using [Microk8s](https://microk8s.io/) include:
+	  - Can be installed on any Linux system as a
+	    [snap](https://snapcraft.io/)
+	  - Strong isolation and update semantics - your single-node cluster
+	    will be updated within a short period after upstream Kubernetes
+	    releases.
+          - GPU pass through built in - e.g. **microk8s.enable gpu**
+      * If you are not on a Linux machine, or you want to use Kubeflow in a
+	confined environment, then use
+	[Multipass](https://github.com/CanonicalLtd/multipass) to launch a
+	virtual machine. Benefits include:
+	  - Ubuntu [cloud-images](http://cloud-images.ubuntu.com/) already
+	    integrated.
+	  - Lightweight hypervisor using native operating system mechanisms
+	    (e.g. [Hypervisor
+	    Framework](https://developer.apple.com/documentation/hypervisor) on
+	    MacOS, [Hyper-V on Windows
+	    10](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v),
+	    QEMU/KVM for linux)
+	  - Eliminates the need to install a separate virtualization
+	    application.
+	  - You can use cloud-init to customize the VM (as you might in a cloud)
 
-This list is based ONLY on current capabilities. We are investing significant resources to expand the
-functionality and actively soliciting help from companies and individuals interested in contributing (see [Contributing](/docs/contributing/)).
+### Cloud
 
-## Set up Kubernetes
+   * [Kubernetes Engine setup](/docs/started/getting-started-gke/).
```

Comment:
Please change this to a sentence instead of a single-item list.

Reply:
Thanks